NSS-Day-Cohort-53/tabloid-full-stack-always-picturesque-icebergs
# added view all tags to the website

#### Changes Made
1. Added TagControler.cs, ` to `Controllers ` directory.
2. Added Tag.cs, ` to `Models` directory.
3.  Added ITagRepository.cs, TagRepository.cs ` to `Repository` directory.
4. added TagMangement.js and TagView.js to Modules directory

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout nc-displayAllTags
    npm start
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   When you first log into the website and run your API you should see Tag Manager in the nav bar. When you click on that it should send you to a new page that shows a list of all the tags that are listed in the datebase.
expected outcome is.
    When user clicks on tag manager, then a new page should show with a list of all the tags.
6. View the code file.
   closes #35
